### DISCUSSION TITLE
Need help in turn off docker in 'AutogenBuilder"

### DISCUSSION THREAD

**Discussion by mraguth** [microsoft/autogen#1484] [General]
I am new and struggling to turn off Docker. I like to either set 'use_docker': false or set up environment virable. Either option will work. Thanks for your help. The screenshot  has been enclosed.
![docker](https://github.com/microsoft/autogen/assets/122765302/8da7bee6-623b-45fa-95c5-aafd3b327745)

**Comment by ekzhu** [comment]
If you are on UNIX system you can set environment variable through `export AUTOGEN_USE_DOCKER=False`. 

On Windows you can edit your environment variables in system settings.

**Reply by sonichi** [reply]
@LinxinS97 @LeoLjl FYI
